Frameworks/UIKit/UIActivityIndicatorView.mm

```diff
@@ -129,57 +133,49 @@ - (instancetype)initWithFrame:(CGRect)frame {
  Microsoft Extension
 */
 - (instancetype)initWithFrame:(CGRect)frame xamlElement:(RTObject*)xamlElement {
-    // TODO: We're passing nil to initWithFrame:xamlElement: because we have to *contain* a _subview for padding/layout.
-    // Note: Pass 'xamlElement' instead, once we move to a *single* backing Xaml element for UIActivityIndicatorView.
-    if (self = [super initWithFrame:frame xamlElement:nil]) {
-        [self _initUIActivityIndicatorView:xamlElement];
+    if (self = [super initWithFrame:frame xamlElement:xamlElement]) {
+        [self _initUIActivityIndicatorView];
     }
 
     return self;
 }
 
 /**
  @Status Interoperable
+ @Note Frame will not be resized smaller than the contained ProgressRing, as this would obscure it.
```

Comment:
So, the more likely case is that someone initializes a UIActivityIndicatorView with a CGRect and either sets the size to the default indicator size, 20x20, or omits the size by just putting 0x0 (since that Just Works™ on iOS). Obviously if it's the second case, it would be immediately obvious. However, the first case would appear fine until they were to set it to the White Large style, which would then resize the indicator to 37x37, obscuring it in XAML but not in iOS.

---
In reply to: [113079902](https://github.com/Microsoft/WinObjC/pull/2608#discussion_r113079902) [](ancestors = 113079902,113078524)